style.scss

```
@@ -0,0 +1,200 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+@mixin position($position) {
+  @if $position == "relative" {
+    position: relative;
+  }
+  @if $position == "title_img" {
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+  @if $position == "price" {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  @if $position == "new_img" {
+    position: absolute;
+    top: 0;
+    transform: translate(-20%, -20%);
+  }
+  @if $position == "new_p" {
+    position: absolute;
+    top: 0;
+    left: 0;
+    transform: translate(150%, -10%);
+  }
+}
+
+@mixin bg($bgcolor) {
+  background-color: $bgcolor;
+}
+@mixin text($font_text) {
+  font-size: $font_text;
+}
+@mixin font($font_family) {
+  font-family: $font_family;
+}
+@mixin leading($line_height) {
+  line-height: $line_height;
+}
+.justify-center {
+  justify-content: center;
+}
+.flex {
+  display: flex;
+}
+.relative {
+  position: relative;
+}
+.absolute {
+  position: absolute;
+}
+.text-center {
+  text-align: center;
+}
+
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 100%;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+.main {
+  flex-wrap: wrap;
+  margin-top: 100px;
+}
+.item {
+  width: 100%;
+ @include position("relative");
+  max-width: 340px;
+  padding: 0px 30px;
+  margin-bottom: 100px;
+  &_content {
```

```suggestion
  &-content {
```
Nếu em đặt tên theo quy tắc BEM thì mới nên dùng dấu _ nhé, còn không thì từ nào dài cứ ngăn cách bởi dấu - cho anh nhé